.github/workflows/test-application.yaml

```diff
@@ -167,3 +167,99 @@ jobs:
+                      phpcr-transport: 'jackrabbit'
+                      dependency-versions: 'highest'
+                      php-extensions: 'ctype, iconv, intl, mysql, pdo_mysql, php_fileinfo, imagick'
+                      tools: 'composer'
+
+        steps:
+            - name: Checkout project
+              uses: actions/checkout@v2
+
+            - name: Install and configure Node.js
+              uses: actions/setup-node@v2
+              with:
+                  node-version: '14'
+
+            - name: Install and configure PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  extensions: ${{ matrix.php-extensions }}
+                  tools: ${{ matrix.tools }}
+                  ini-values: 'memory_limit=-1'
+
+            - name: Install and configure MySQL
+              uses: shogo82148/actions-setup-mysql@v1.0.1
+              with:
+                  mysql-version: ${{ matrix.mysql-version }}
+                  root-password: 'root'
+                  my-cnf: |
+                      [mysqld]
+                      default-authentication-plugin=mysql_native_password
+
+            - name: Cache jackrabbit runnable
+              uses: actions/cache@v2
+              id: jackrabbit-cache
+              if: ${{ matrix.phpcr-transport == 'jackrabbit' }}
+              with:
+                  path: downloads
+                  key: ${{ runner.os }}-jackrabbit-${{ env.JACKRABBIT_VERSION }}
+
+            - name: Download jackrabbit
+              if: ${{ matrix.phpcr-transport == 'jackrabbit' && steps.jackrabbit-cache.outputs.cache-hit != 'true' }}
+              run: |
+                  mkdir downloads
+                  cd downloads
+                  curl -O http://archive.apache.org/dist/jackrabbit/${{ env.JACKRABBIT_VERSION }}/jackrabbit-standalone-${{ env.JACKRABBIT_VERSION }}.jar
+                  cd -
+
+            - name: Run jackrabbit
+              if: ${{ matrix.phpcr-transport == 'jackrabbit' }}
+              run: powershell -command "Start-Process -FilePath java -ArgumentList '-jar downloads/jackrabbit-standalone-${{ env.JACKRABBIT_VERSION }}.jar'"
+
+            - name: Require jackrabbit dependencies
+              if: ${{ matrix.phpcr-transport == 'jackrabbit' }}
+              run: |
+                  composer remove jackalope/jackalope-doctrine-dbal --dev --no-interaction --no-update
+                  composer require jackalope/jackalope-jackrabbit:^1.4  --no-interaction --no-update
+
+            - name: Install composer dependencies
+              uses: ramsey/composer-install@v1
+              with:
+                  dependency-versions: ${{ matrix.dependency-versions }}
+
+            - name: Bootstrap test environment
+              run: composer bootstrap-test-environment
+
+            - name: Execute test cases
+              run: composer test
+
+            - name: Install javascript dependencies
```

Comment:
not sure if this is compatible with windows, but we used the `bahmutov/npm-install` action in the other workflow:
https://github.com/sulu/sulu/blob/8f6c63ab7e4e606e3fea04c1d7e1dfc9b738c1b5/.github/workflows/test-application.yaml#L34-L37